llvm/include/llvm/Analysis/MemorySSAUpdater.h

```diff
@@ -276,7 +276,9 @@ class MemorySSAUpdater {
   // representation for. No other cases are supported.
   void cloneUsesAndDefs(BasicBlock *BB, BasicBlock *NewBB,
```

Comment:
Would probably b good to document the new argument?

Reply:
Done. Also converted the whole comment into doxygen format.